.idea/lynx-web.iml

```diff
@@ -1,9 +0,0 @@
-<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Maybe fixup this commit into a89d68f2c079bd6d75d75778e7bea0d72c7bf608 so that the add and delete of these files won't be shown in the history? It's unrelated to the code written, so I don't think it is valuable to include.

Reply:
I don't know how to do that so I just force push. Lmk if the gitignore template looks fine.